utils/results/conversion/simplejsonparser/simplejsonparser.go

```diff
@@ -0,0 +1,607 @@
+	return func(violation services.Violation, cves []formats.CveRow, applicabilityStatus jasutils.ApplicabilityStatus, severity severityutils.Severity, impactedPackagesName, impactedPackagesVersion, impactedPackagesType string, fixedVersion []string, directComponents []formats.ComponentRow, impactPaths [][]formats.ComponentRow) error {
+		violationOpRiskData := getOperationalRiskViolationReadableData(violation)
+		for compIndex := 0; compIndex < len(impactedPackagesName); compIndex++ {
+			operationalRiskViolationsRow := &formats.OperationalRiskViolationRow{
+				ImpactedDependencyDetails: formats.ImpactedDependencyDetails{
+					SeverityDetails:           severityutils.GetAsDetails(severity, applicabilityStatus, pretty),
+					ImpactedDependencyName:    impactedPackagesName,
+					ImpactedDependencyVersion: impactedPackagesVersion,
+					ImpactedDependencyType:    impactedPackagesType,
+					Components:                directComponents,
+				},
+				IsEol:         violationOpRiskData.isEol,
+				Cadence:       violationOpRiskData.cadence,
+				Commits:       violationOpRiskData.commits,
+				Committers:    violationOpRiskData.committers,
+				NewerVersions: violationOpRiskData.newerVersions,
+				LatestVersion: violationOpRiskData.latestVersion,
+				RiskReason:    violationOpRiskData.riskReason,
+				EolMessage:    violationOpRiskData.eolMessage,
+			}
+			*operationalRiskViolationsRows = append(*operationalRiskViolationsRows, *operationalRiskViolationsRow)
+		}
+		return nil
+	}
+}
+
+func PrepareSimpleJsonLicenses(target string, licenses []services.License) ([]formats.LicenseRow, error) {
+	var licensesRows []formats.LicenseRow
+	err := results.PrepareLicenses(target, licenses, addSimpleJsonLicense(&licensesRows))
+	return licensesRows, err
+}
+
+func addSimpleJsonLicense(licenseViolationsRows *[]formats.LicenseRow) results.PrepareLicensesFunc {
+	return func(license services.License, impactedPackagesName, impactedPackagesVersion, impactedPackagesType string, directComponents []formats.ComponentRow, impactPaths [][]formats.ComponentRow) error {
+		*licenseViolationsRows = append(*licenseViolationsRows,
+			formats.LicenseRow{
+				LicenseKey:  license.Key,
+				ImpactPaths: impactPaths,
+				ImpactedDependencyDetails: formats.ImpactedDependencyDetails{
+					ImpactedDependencyName:    impactedPackagesName,
+					ImpactedDependencyVersion: impactedPackagesVersion,
+					ImpactedDependencyType:    impactedPackagesType,
+					Components:                directComponents,
+				},
+			},
+		)
+		return nil
+	}
+}
+
+func PrepareSimpleJsonJasIssues(target string, entitledForJas, pretty bool, jasIssues ...*sarif.Run) ([]formats.SourceCodeRow, error) {
+	var rows []formats.SourceCodeRow
+	err := results.PrepareJasIssues(target, jasIssues, entitledForJas, func(run *sarif.Run, rule *sarif.ReportingDescriptor, severity severityutils.Severity, result *sarif.Result, location *sarif.Location) error {
```

Comment:
I suggest avoiding inline functions when they're not small and trivial. We can create a function and just pass it's name